task_01.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- conding: utf-8 -*-
+"""Fahrenheit to Celsius Calculator"""
+
+def fahrenheit_to_celsuis(degrees):
```

Comment:
mispelled function name which is why the error message read `'module' object has no attribute 'fahrenheit_to_celsius'`